client/src/components/index.js

```diff
@@ -0,0 +1,8 @@
+// login page
+//--components
+//  --index.js
+//  --header.js
+//  --footer.js
+//  --start.js
+//  --menu.js
+//  --dailyLog
```

Comment:
is dailyLog missing the .js extension?

Reply:
Not in the file title itself, only within index.js when I typing notes while connie and I were meeting.

Reply:
thanks for clarification